starfyre/compiler.py

```diff
@@ -32,10 +32,28 @@ def resolve_css_import(css_file_name, working_directory):
         for line in import_file.readlines():
             css_content.append(line)
 
-    return css_content           
+    return css_content
 
 
-def parse(fyre_file_name): 
+def check_import_line(line, project_dir):
+    
+    """
+    Check if the given line starts with an fyre import statement from the specified project directory.
+
+    Args:
+        line (str): The line to check.
+        project_dir (str): The project directory name.
+
+    Returns:
+        bool: True if the line starts with the specified import statement, False otherwise.
+    """
+    project_dir_name = str(project_dir).split("/")[-1]
+    pattern = rf'^from\s+{project_dir_name}\.' # checks is we have a line like 'from {project_dir}.'
+    match = re.match(pattern, line)
+    return match is not None
```

Comment:
This function's job is to check if a line is an import statement related to importing `fyre` components. This is the first step to resolving absolute import style.